icicle/tests/test_field_api.cpp

```diff
@@ -11,6 +11,9 @@
 #include "icicle/utils/log.h"
 #include "icicle/backend/ntt_config.h"
 
+#include "icicle/program/symbol.h"
+#include "icicle/program/program.h"
+#include "../../icicle/backend/cpu/include/cpu_program_executor.h"
```

Comment:
maybe we should move tests like this one to the backend? this is kind of dirty and works for CPU only anyway.
If we define another test executable in the backend then we will test them only when the backend is compiled and avoid this uglyness

Reply:
we talked about that yesterday
We said we'll start with puting it in field and see what is the volume.
Add another suite for backend is fine

Reply:
I meant sumcheck in field, not directly testing backends here

Reply:
to be resolved once Aviad will make order in the CI and the test suits